### PR TITLE
Update kube-cross to v1.16.4-2

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

This patch bumps kube-cross to the latest release.

#### Which issue(s) this PR fixes:

Refers to https://github.com/kubernetes/k8s.io/pull/2105

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Review by xmudrii — APPROVED** [review]

**Comment by saschagrunert** [comment]
/assign @dims 
for sample apiserver approval

File: test/images/sample-apiserver/Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ export
 # Get without building to populate module cache
 # Then, get with OS/ARCH-specific env to build
 bin:
-	docker run --rm -i -v "${TARGET}:${TARGET}:Z" k8s.gcr.io/build-image/kube-cross:v1.16.4-1 \
+	docker run --rm -i -v "${TARGET}:${TARGET}:Z" k8s.gcr.io/build-image/kube-cross:v1.16.4-2 \
```
Comment (saschagrunert) [inline test/images/sample-apiserver/Makefile:27]: I think if we make this a `$(cat ...)` command for `VERSION` then we would not have to request approval for every kube-cross build.

Comment (cpanato) [reply test/images/sample-apiserver/Makefile:27]: +1

Comment (dims) [reply test/images/sample-apiserver/Makefile:27]: +1 from me @saschagrunert

**Review by dims — APPROVED** [review]
/lgtm

**Comment by saschagrunert** [comment]
/assign @justaugustus 
for build/OWNERS approval

**Comment by justaugustus** [comment]
/lgtm
/approve

+1 to parameterizing the kube-cross version

**Pull request merged**